docs/errors/E0327.md

```diff
@@ -0,0 +1,21 @@
+# E0327: 'function async f()' is not allowed; write 'async function f()' instead
+
+Class methods can have modifiers such as `export`, `static`, `async`, and `*`. It is a
+syntax error to write `function async f()` in a function declaration because async will
+not be recognized as a keyword but as the function name:
```

Comment:
> async will not be recognized as a keyword but as the function name

Well, this isn't true, because it's a syntax error. I'd remove this part of the sentence.

I suggest this wording instead for the opening paragraph:

> Functions can be marked with the `async` keyword. It is a syntax error to write the `async` keyword after the `function` keyword:

Reply:
I get you want to tell users the fun fact that `async` can be a function name, and that the error is really two identifiers back-to-back; but I think teaching users this will just further confuse them.  sTRAGER's suggestion is better.  My suggestion is best

> Functions can be marked using the `async` keyword like `async function f() {`.  It is a syntax error to write the `async` keyword after the `function` keyword:

Reply:
Thank you both, much clearer like that.